docs: identity mapper (#293)
flanksource/docs: mission-control/docs/installation/self-hosted/oidc.mdx

                              microsoft_tenant: # The Azure AD Tenant Id
                              client_id: #...
                              client_secret: #...
                              mapper_url: base64:// #base64 encoded jsonnet schema
                              scope:
                                - email
                                - openid
                                - profile
    ```
    <p/>

5. Optionally, create a cel expression to map identities from the OIDC provider to a mission control role & team. 

>     _Example_: the following script maps all Azure users in the "SRE" group to the "admin" role & everyone else to a "viewer" role.

Reviewer: ```suggestion
    The following script maps all Azure users in the `SRE` group to the `admin` role and everyone else to the `viewer` role.
```

Reviewer: Might need to list roles somewhere

    ```yaml
    apiVersion: v1
    kind: ConfigMap                 
    metadata:                       
      name: azure-identity-mapper
    data:
      script: >
        {  
          "role": "sre" in identity.traits.groups ?  "admin":  "viewer"
        }
    ```
    <p/>

>     The cel expression is expected to return an object with a `role` & a `teams[]` fields.

Reviewer: But this in a table

Reviewer: I think we also need a reference for the identity object
